src/components/profile/MyProfile.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import ApptTicker from './ApptTicker';
+import MyBarbers from './MyBarbers';
+import MyCuts from './MyCuts'
+import MyBookmarks from './MyBookmarks';
+
+
+class MyProfile extends Component {
+    state = {
+        myBarbersView: true,
+        myCutsView: false,
+        myBookmarksView: false
+    };
```

Comment:
not sure how i feel about storing three separate boolean values, but we can change this later